src/test/scala/org/zalando/kanadi/SubscriptionsSpec.scala

```diff
@@ -35,28 +31,19 @@ class SubscriptionsSpec(implicit ec: ExecutionEnv) extends Specification with Co
     EventTypes(nakadiUri, None)
   val subscriptionIds: mutable.ParSet[SubscriptionId] = mutable.ParSet.empty
 
-  eventTypeName.pp
-  s"Consumer Group: $consumerGroup".pp
+  pp(eventTypeName)
+  pp(s"Consumer Group: $consumerGroup")
 
   def createEventType = eventsTypesClient.create(EventType(eventTypeName, OwningApplication, Category.Business))
 
-  override def beforeAll =
+  override def beforeAll(): Unit = {
     Await.result(createEventType, 10 seconds)
-
-  override def afterAll = {
```

Comment:
I ended up deleting this method because it wasn't actually being run correctly and when you I fixed it I got

```
[info] org.zalando.kanadi.SubscriptionsSpec *** ABORTED ***
[info]   org.zalando.kanadi.models.GeneralError: Error from server, response is Problem(None,Conflict,409,Some(Can't remove event type Kanadi-Test-Event-be9f9995-7b57-494f-a09f-64355b16d73e, as it has subscriptions),None)
```

In reality this cleanup is an artifact of when Kanadi used to test against the actual Zalando test realm (where you did need to clean up subscriptions), and since we run against a docker image this is pointless now